certified-connectors/InEight/readme.md

```diff
@@ -0,0 +1,39 @@
+# InEight Connector
+InEight Connector provides an API to work with InEight objects.
```

Comment:
@InEightDevOps Please provide a detailed description of your connector that can be understandable by everyone.